frontend/src/components/pages/ViewDonations.tsx

```diff
@@ -47,7 +47,7 @@ const ViewDonations = ({
   };
 
   return (
-    <Container alignContent="left" variant="calendarContainer">
+    <Container alignContent="left" variant="responsiveContainer">

```

Comment:
can we revert this change? Calendar isn't aligned otherwise: 
![image](https://user-images.githubusercontent.com/19617248/158503815-834ea30f-734e-4daa-8317-a3760b766602.png)